lib/classes/content/BulletedList.js

```diff
@@ -0,0 +1,35 @@
+class BulletedList extends Array {
+	/**
+	 * List marker.
+	 * @type {string}
+	 */
+	#marker = "+";
+
+	/**
+	 * @param {string} value New marker value.
+	 */
+	set marker(value) {
+		if (typeof value === "string" && value.length > 0)
```

Comment:
Is type checking really needed here?

Reply:
Just for safety. This is not **that** important just because any value can be converted to string.

Reply:
We can also set some warning messages on trying to set non-string value as a marker.

Reply:
But what if we want to set empty string as value? Or a number? It seems to me that such checks are not needed.

Reply:
This is not an ordered list, but you can, you know, set the string with numeric symbols. Or if you want to set an empty string as marker then this class is not useful for you. This is bulleted list and it must use something as a marker. Anyway, if you want, I can just convert any values passed to `marker` to string type. And after that you will allow you use `null`, `undefined` and `[object Object]` as a marker.

Reply:
So... In this case add a check for `null`, `undefined` and `[object Object]`, and then return the default value instead as an option?
The problem here in non-strongly typed languages: we are waiting for the value, but we do not know what type it is.

Reply:
Since there only `string`, `number` and `object` types available I'll just add numbers to allowed marker type.